include/cpuinfo_x86.h

```diff
@@ -119,7 +119,10 @@ CacheInfo GetX86CacheInfo(void);
 
 typedef enum {
   X86_UNKNOWN,
-  ZHAOXIN,           // ZX
+  ZHAOXIN_ZHANGJIANG,  // ZhangJiang
+  ZHAOXIN_WUDAOKOU,  //WuDaoKou
+  ZHAOXIN_LUJIAZUI,  //LuJiaZui
+  ZHAOXIN_YONGFENG,  //YongFeng
```

Comment:
nit: space after `//` like for the dfirst one

@gchatelet 
ref:
https://en.wikichip.org/wiki/zhaoxin/microarchitectures/zhangjiang
https://en.wikichip.org/wiki/zhaoxin/microarchitectures/wudaokou
https://en.wikichip.org/wiki/zhaoxin/microarchitectures/lujiazui

did found the YongFeng...

Reply:
also did not find any information about YongFeng

Reply:
YongFeng  is our latest product and will be introduced to the market soon. We hope this submission can support this latest product.
I will add space after //.